packages/aleo-types/src/address.rs

```diff
@@ -4,20 +4,24 @@ use std::str::FromStr;
 use axelar_wasm_std::nonempty;
 use cosmwasm_std::HexBinary;
 use error_stack::{ensure, Report, Result};
+use lazy_static::lazy_static;
 use serde::{Deserialize, Serialize};
 
 use crate::{verify_becnh32, Error};
 
 pub const ALEO_ADDRESS_LEN: usize = 63;
+lazy_static! {
+    pub static ref ZERO_ADDRESS: Address = Address::default();
+}
```

Comment:
No need to introduce `lazy_static!` as a dependency here:
[`std::sync::LazyLock`](https://doc.rust-lang.org/std/sync/struct.LazyLock.html) can do the same. 

```rust
use std::sync::LazyLock;
pub static ZERO_ADDRESS: LazyLock<Address> = LazyLock::new(|| Address::default());
```